src/modules/admin/wrappers/admin-core-config.tsx

```diff
@@ -203,6 +203,7 @@ export function getAdminCoreConfig(router: NextRouter | null, locale: Locale): A
 			},
 		},
 		routes: ADMIN_CORE_ROUTES_BY_LOCALE[locale],
+		locale: locale as any,

```

Comment:
Avoid `any` if possible

Reply:
yeah, i tried, but the Locale from admin-core doesn't seem to be compatible with the Locale enum from the client :s